src/main/java/graphql/nadel/instrumentation/NadelInstrumentation.java

```diff
@@ -27,6 +29,7 @@
  * one for the step is `dispatched` and one for when the step has `completed`.  This is done because many of the "steps" are asynchronous
  * operations such as fetching data and resolving it into objects.
  */
+@SuppressWarnings("unused")
 public interface NadelInstrumentation {
```

Comment:
Is this going to support field instrumentation in the future? Or do we have a different plan? This would be needed to support Apollo  tracing

Reply:
So field tracing in Nadel / Gateway is interesting.

We dont visit all the sub fields - only the ones we invoke directly or via hydration

So should it trace just them or all fields.  The invoked calls are the most useful but all fields give usage visibility